From: Boyuan Yang <byang@debian.org>
Date: Mon, 11 Jan 2021 18:18:16 -0500
Subject: Makefile: allow env buildflags

---
 Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 00d1c0f..018a139 100644
--- a/Makefile
+++ b/Makefile
@@ -3,12 +3,12 @@
 # Copyright (C) 2001, 2009 Gergely Risko
 # Can be licensed under the terms of GPL v3 or above
 
-CC=gcc
-CFLAGS=-Wall -g -O2 -pedantic
-LDFLAGS=-g
+CC ?= gcc
+CFLAGS ?= -Wall -g -O2 -pedantic
+LDFLAGS ?= -g
 LIBS=-lncurses
 OBJS=bombardier.o display.o date.o randomhouse.o step.o hof.o signal.o gcurses.o
-DESTDIR=/
+DESTDIR ?= /
 
 all: bombardier
 
